cinema/serializers.py

```diff
@@ -114,9 +114,9 @@ class TicketSerializer(serializers.ModelSerializer):
     def validate(self, attrs):
         data = super(TicketSerializer, self).validate(attrs=attrs)
         Ticket.validate_ticket(
-            attrs["row"], 
-            attrs["seat"], 
-            attrs["movie_session"].cinema_hall, 
+            attrs["row"],
+            attrs["seat"],
+            attrs["movie_session"].cinema_hall,
             ValidationError
         )
```

Comment:
The `validate_ticket` method is being called with `ValidationError` as an argument, which seems incorrect. Ensure that `validate_ticket` is a method of the `Ticket` model and that it accepts the arguments being passed. If `ValidationError` is intended to be used for raising exceptions, it should be handled within the method.